README.md

```diff
@@ -59,16 +59,16 @@ Lightning Transformers tasks allow you to train models using HuggingFace Transfo
-|                                                                                                                     |                                                                                                                          | <img width=200/>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Task                                                                                                                | Quick Commands                                                                                                                    | Run                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
what is the change here?

Reply:
as we are not going to use `trainer.gpus` the link should also reflect that, so I changed the links `train.py&cloud=grid&instance=p3.2xlarge&accelerators=1&disk_size=200&framework=lightning&script_args=train.py+task=nlp/translation+dataset=nlp/translation/wmt16+trainer.gpus=1` from this

to 

`train.py&cloud=grid&instance=p3.2xlarge&accelerators=1&disk_size=200&framework=lightning&script_args=train.py+task=nlp/translation+dataset=nlp/translation/wmt16+trainer.accelerator=gpus`